### PR TITLE
Added small improvements.

### PR DESCRIPTION
**Pull request by MarcinMaciaszek**
Small improvements in library

I have updated setup.py reqs and library scope. SUITE scope make much more sense for this library. 

Fixes #no_issue

Before merge following needs to be applied:
- [ ] At least one example testcase added in testcases/ --
- [ ] Library Documentation regenerated according to [Generate docs](https://github.com/devopsspiral/KubeLibrary#generate-docs)
- [x] PR entry added in CHANGELOG.md in **In progress** section
- [ ] All new testcases tagged as **prerelease** along other tags to exclude it from execution until released on PyPI
- [ ] Coverage threshold increased in [.coveragerc](https://github.com/devopsspiral/KubeLibrary/blob/master/.coveragerc) if new coverage is higher than actual, see the lint-and-coverage step in CI
```
fail_under = 67
```

### CODE REVIEW

**Comment by MarcinMaciaszek** [comment]
@Nilsty @m-wcislo could you please review this PR?

File: .circleci/config.yml [hunk 1]
```diff
@@ -11,7 +11,9 @@ jobs:
     steps:
       - checkout
       - python/load-cache
-      - python/install-deps
+      - run:
```
Comment (m-wcislo) [inline .circleci/config.yml:14]: We could use https://circleci.com/developer/orbs/orb/circleci/python?version=0.3.2#commands-install-deps with specified file name

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## In progress
+### Changed
+- Refactored setup.py and requirements
```
Comment (m-wcislo) [inline CHANGELOG.md:9]: Please keep it as one line with link to PR and author

File: src/KubeLibrary/version.py [hunk 3]
```diff
@@ -0,0 +1 @@
+version = "0.6.1"
```
Comment (m-wcislo) [inline src/KubeLibrary/version.py:1]: we can keep it as 0.6.0, it is incremented just before release.

**Review by m-wcislo — CHANGES_REQUESTED** [review]
Nice work, I really like making it less messy! I've added some small comments.

**Review by m-wcislo — APPROVED** [review]

**Pull request merged**